Plots/Contours/NCL_conwomap_2.py

```diff
@@ -0,0 +1,78 @@
+"""
+NCL_conwomap_2.py
+===============
+This script illustrates the following concepts:
+   - Drawing a simple filled contour plot
+   - Selecting a different color map
+   - Changing the size/shape of a contour plot using viewport resources
+See https://www.ncl.ucar.edu/Applications/Scripts/conwomap_2.ncl for further information.
+"""
+
+###############################################################################
+# Import packages
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+from util.make_byr_cmap import make_byr_cmap
+
+import matplotlib.pyplot as plt
+import matplotlib.ticker as tic
+
+###############################################################################
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset("../../data/netcdf_files/cone.nc")
+u = ds.u.isel(time=4)
+
+###############################################################################
+# Plot
+# First get axes for a projection of preference
+fig = plt.gcf()
```

Comment:
```suggestion
fig = plt.figure()
```
make it clear that we are making a figure. `gcf` does that but is less clear I think.